django/db/backends/base/schema.py

```diff
@@ -97,6 +99,7 @@ def execute(self, sql, params=()):
                 "that can't perform a rollback is prohibited."
             )
         # Log the command we're running, then run it
+        sql = force_text(sql)
```

Comment:
I just wanted to account for the case where Python 2 `str()` ended up here as well. Since master is now Python 3 I'll convert all the `force_text()` calls I added to `str()`.